lms/djangoapps/grades/management/commands/compute_grades.py

```diff
@@ -108,15 +108,11 @@ def _shuffled_task_kwargs(self, options):
         all_args = []
         estimate_first_attempted = options['estimate_first_attempted']
         for course_key in self._get_course_keys(options):
-            enrollment_count = CourseEnrollment.objects.filter(course_id=course_key).count()
-            if enrollment_count == 0:
-                log.warning("No enrollments found for {}".format(course_key))
-            batch_size = self._latest_settings().batch_size if options.get('from_settings') else options['batch_size']
-            for offset in six.moves.range(options['start_index'], enrollment_count, batch_size):
-                # This is a tuple to reduce memory consumption.
-                # The dictionaries with their extra overhead will be created
-                # and consumed one at a time.
-                all_args.append((six.text_type(course_key), offset, batch_size))
+            # This is a tuple to reduce memory consumption.
+            # The dictionaries with their extra overhead will be created
+            # and consumed one at a time.
+            for task_arg in tasks._course_task_args(course_key, **options):
```

Comment:
very small change, but sort of important: the items being generated by `_course_task_args()` are a tuple of args, so `task_arg` should really be plural (same thing for the line right below this).